src/pages/about.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import Layout from '../components/layout';
+import ContactParagraph from '../components/ContactParagraph';
+import AboutPageImage from '../components/AboutPageImage';
+
+import {
+    aboutPageContainer,
+    aboutPageText,
+} from '../styles/aboutPage.module.scss';
+
+const AboutPage = () => {
+    return (
+        <Layout>
+            <AboutPageImage/>
+            <h1>About us</h1>
+            <div className={aboutPageContainer}>
+                <p className={aboutPageText}>
+                    Increasingly, software development tools and platforms stand out for the Developer Experience they promote. The way software development is organised is also started to get recognised as a factor contributing to Developer Experience. Developers intuitively know what they like and what they don't like, but it can be hard to pin down why. Designing for good Developer Experience is not very common, and how it links to user, company, and societal goals is not well understood. We think Developer Experience should be examined in scientific research to gain a greater understanding of what drives good experiences for developers, how to design for good Developer Experience, and how to link it with experiences that others have.
```

Comment:
Instead of having this as one paragraph with forced line breaks, how about making it three paragraphs?